app/api/users.py

```
@@ -0,0 +1,46 @@
+from fastapi import APIRouter, Depends, HTTPException, status
+from sqlalchemy.ext.asyncio import AsyncSession
+from fastapi.security import OAuth2PasswordBearer, OAuth2PasswordRequestForm
+from app.core.database import get_db
+from app.schemas import UserCreate, UserLogin, UserResponse, Token
```

Flake8 (7.2.0)

[error] 'app.schemas.UserLogin' imported but unused

(F401)